shipyard/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "",
+        "image_url": "images/datadog-input.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "",
+        "image_url": "images/shipyard-teal.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
Please add relevant categories and submitted data from [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)

Reply:
Thanks for the link I have added relevant classifier tags here: 9526ff0b2936103419beb37f9d845451ac936117